doc/select.md

```diff
@@ -202,11 +202,62 @@ crud.select('developers', {{'==', 'surname', "Adams"}})
 
 ### Avoiding full scan
 
+Most requests lead to a full scan. A critical log entry containing the current stack traceback is created upon such calls with a message: `Potentially long select from space '%space_name%'`.
+
 **Example:**
 
 ```lua
 crud.select('developers', {{'==', 'surname', "Adams"}, {'>=', 'age', 25}})
 ---
+2022-05-24 14:06:31.748 [25108] main/103/playground.lua C> Potentially long select from space 'developers'
+ stack traceback:
+	.rocks/share/tarantool/crud/select/compat/common.lua:24: in function 'check_select_safety'
+	.rocks/share/tarantool/crud/select/compat/select.lua:298: in function <.rocks/share/tarantool/crud/select/compat/select.lua:252>
+	[C]: in function 'pcall'
+	.rocks/share/tarantool/crud/common/sharding/init.lua:163: in function <.rocks/share/tarantool/crud/common/sharding/init.lua:158>
+	[C]: in function 'xpcall'
+	.rocks/share/tarantool/errors.lua:145: in function <.rocks/share/tarantool/errors.lua:139>
+	[C]: in function 'pcall'
+	builtin/box/console.lua:403: in function 'eval'
+	builtin/box/console.lua:709: in function 'repl'
+	builtin/box/console.lua:758: in function 'start'
+	doc/playground.lua:164: in main chunk
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucket_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'surname', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [3, 9661, 'Pavel', 'Adams', 27]
+...
+```
+
+You can avoid the full scan with '=' or '==' condition on an index fields.
```

Comment:
Nit: an index fields -> index fields (plural).

Also the order of conditions is important. I would highlight it.

Reply:
As I understood, the following sentence (it is removed from the select.md document) is just incorrect:

> In this case, a full scan will be performed, since non-indexed field is placed first in search conditions. Example below shows how you can avoid a full scan.

I can add a condition on a non-indexed field as the first condition and it will not change anything about the query plan (we'll not go to the linear scanning). Correct?

Reply:
Yes, non-index conditions don't change anything. Only a first condition on index fields matters.